source/crud/crud.rst

```diff
@@ -11,8 +11,8 @@ Driver CRUD API
 :Advisors: Jeremy Mikola, Jeff Yemin
 :Status: Approved
 :Type: Standards
-:Minimum Server Version: 2.6
-:Last Modified: October 28, 2019
+:Minimum Server Version: 2.7
```

Comment:
I don't think this line should be modified. If we do need to raise this value, such a change likely belongs in its own ticket.

Reply:
Good catch. I thought I was incrementing the document version!

Reply:
Done